### PR TITLE
mon: move 'pg map' to OSDMonitor

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: src/mon/OSDMonitor.cc [hunk 1]
```diff
@@ -3823,6 +3823,50 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
...
+    pg_t mpgid = osdmap.raw_pg_to_pg(pgid);
+    osdmap.pg_to_up_acting_osds(pgid, up, acting);
+    if (f) {
+      f->open_object_section("pg_map");
+      f->dump_unsigned("epoch", osdmap.get_epoch());
+      f->dump_stream("raw_pgid") << pgid;
+      f->dump_stream("pgid") << mpgid;
+
+      f->open_array_section("up");
+      for (vector<int>::iterator it = up.begin(); it != up.end(); ++it)
```
Comment (tchaikov) [inline src/mon/OSDMonitor.cc]: could use range-based loop.

File: src/mon/OSDMonitor.cc [hunk 2]
```diff
@@ -3823,6 +3823,50 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
...
+	f->dump_int("up_osd", *it);
+      f->close_section();
+
+      f->open_array_section("acting");
+      for (vector<int>::iterator it = acting.begin(); it != acting.end(); ++it)
```
Comment (tchaikov) [inline src/mon/OSDMonitor.cc]: ditto.

File: qa/tasks/ceph_manager.py [hunk 3]
```diff
@@ -1297,24 +1297,20 @@ def get_pg_replica(self, pool, pgnum):
...
         """
-        output = self.raw_cluster_cmd("pg", "dump", '--format=json')
-        j = json.loads('\n'.join(output.split('\n')[1:]))
         pg_str = self.get_pgid(pool, pgnum)
-        for pg in j['pg_stats']:
-            if pg['pgid'] == pg_str:
-                return int(pg['acting'][0])
+        output = self.raw_cluster_cmd("pg", "map", pg_str, '--format=json')
+        j = json.loads('\n'.join(output.split('\n')[1:]))
+        return int(pg['acting'][0])
```
Comment (tchaikov) [inline qa/tasks/ceph_manager.py]: @liewegas 
`s/pg/j/`

**Comment by liewegas** [comment]
@tchaikov fixed

**Pull request merged**